src/hvps/devices/caen/channel.py

```diff
@@ -159,7 +159,7 @@ def imrange(self) -> bool:
         )
         if response not in ["HIGH", "LOW"]:
             raise ValueError(f"Unexpected response {response}")
-        return response == "HIGH"
+        return response
```

Comment:
Property signature annotation return type should be updated from `bool` to `str`. (Don't worry I will update it after merge since it has no practical effect).